src/rocks.ts

```diff
@@ -0,0 +1,206 @@
+import { ChronerQuest, ChronerStrategy } from "./engine";
+import Macro from "./macro";
+import { chooseQuestOutfit, ifHave } from "./outfit";
+import {
+  familiarWeight,
+  handlingChoice,
+  myAscensions,
+  runChoice,
+  toUrl,
+  use,
+  visitUrl,
+} from "kolmafia";
+import {
+  $familiar,
+  $item,
+  $items,
+  $location,
+  get,
+  getKramcoWandererChance,
+  set,
+} from "libram";
+
+const location = $location`The Cave Before Time`;
+export const bigRock: ChronerQuest = {
+  name: "Get Rock",
+  location,
+  tasks: [
+    {
+      name: "getHit",
```

Comment:
I think a lot of these tasks should be renamed--it's outward-facing content that's worth having a clear, descriptive name